test/model.test.ts

```diff
@@ -21,6 +21,23 @@ describe('SpecQueryModel', () => {
     return SpecQueryModel.build(specQ, schema, DEFAULT_QUERY_CONFIG);
   }
 
+  describe('getDefaultName', () => {
+    it('should have no duplicate default names.', () => {
+      let props: Property[] = DEFAULT_PROPERTY_PRECENCE;
+
+      let unique = props.map((prop) => {
```

Comment:
Why do you need to count?   Why don't just assert that each key added should not already exist?